packages/realm-server/migrations/1728561353454_create-user-billing.js

```diff
@@ -0,0 +1,128 @@
+exports.up = (pgm) => {
+  pgm.createTable('users', {
+    id: 'id',
+    matrix_user_id: { type: 'varchar', notNull: true, unique: true },
+    stripe_customer_id: { type: 'varchar' },
+    created_at: {
+      type: 'timestamp',
+      notNull: true,
+      default: pgm.func('current_timestamp'),
+    },
+  });
+
+  pgm.createTable('plans', {
+    id: 'id',
+    name: { type: 'varchar', notNull: true },
+    monthly_price: { type: 'numeric', notNull: true },
+    credits_included: { type: 'integer', notNull: true },
+    overage_fee_per_credit_in_usd: { type: 'numeric' },
+    created_at: {
+      type: 'timestamp',
+      notNull: true,
+      default: pgm.func('current_timestamp'),
+    },
+  });
+
+  pgm.createTable('billing_cycles', {
+    id: 'id',
+    subscription_id: { type: 'integer' },
+    period_start: { type: 'timestamp', notNull: true },
+    period_end: { type: 'timestamp', notNull: true },
```

Comment:
so when stripe sends us this webhook for a subscription renewal, is it per user? if so shouldn't we capture that column as well?

Reply:
oh wait never mind-- I just noticed the next table.,..